db/migrate/20220811225830_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
```

Comment:
Since we are only using only `username` field to authenticate a user, I would suggest you comment out the e-mail and password fields since they will not be required. Thank you.